core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -368,7 +368,7 @@ protected synchronized <V> V retryRPC(RetryPolicy retryPolicy, RpcCallable<V> rp
     // TODO(binfan): create RPC context so we could get RPC duration from metrics timer directly
     long startMs = System.currentTimeMillis();
     logger.debug("Enter: {}({})", rpcName, debugDesc);
-    try (Timer.Context ctx = MetricsSystem.timer(getQualifiedMetricName(rpcName)).time()) {
+    try (Timer.Context ignored = MetricsSystem.timer(getQualifiedMetricName(rpcName)).time()) {
```

Comment:
I noticed you changed all the "try-with-resources" variable names to `ignored`. It's true that we don't reference the variable anywhere but it's also semantically not quite true because we _are_ actually using the resource (implicitly).

Not sure if this is the style we want to be using, as if there's a "try-with-resources" with more than one variable then how would this pattern scale? `ignored1`, `ignored2`, etc. seems worse to me than just having the variable name as before.

Reply:
I also found this pattern a little strange. IntelliJ was showing a warning about the naming and suggested it be renamed to `ignored` to remove that warning. I could reverse it if we believe it makes the code less legible.

Reply:
IMO let's just let IntelliJ complain on this one, but maybe get a second opinion before you revert all that.

Reply:
I reverted it.